scripts/vim-patch.sh

```diff
@@ -132,37 +132,54 @@ get_vim_patch() {
 
   # Patch surgery: preprocess the patch.
   #   - transform src/ paths to src/nvim/
-  local vim_full="$(git show -1 --pretty=medium "${vim_commit}" \
+  local vim_full
+  vim_full="$(git show -1 --pretty=medium "${vim_commit}" \
     | LC_ALL=C sed -e 's/\( [ab]\/src\)/\1\/nvim/g')"
   local neovim_branch="${BRANCH_PREFIX}${vim_version}"
 
   cd "${NEOVIM_SOURCE_DIR}"
-  local git_remote=$(find_git_remote)
-  local checked_out_branch="$(git rev-parse --abbrev-ref HEAD)"
+  local git_remote
+  git_remote=$(find_git_remote)
```

Comment:
Maybe quotes here? Though I do not think they are really needed, this makes it more consistent with the surrounding code.